src/pages/search.js

```diff
@@ -0,0 +1,14 @@
+import Head from 'next/head';
+
+export default function Search() {
+	return (
+		<>
+			<Head>
+				<title key="title">Search</title>
+				<meta key="description" name="description" content="About" />
+			</Head>
+			<h1>Search</h1>
+			<input placeholder="Where do you want to search"></input>
+		</>
```

Comment:
Your input element is missing a label. A placeholder text is not a suitable substitute for a label element and should always be consider as extra. There exists also the recommendation to not use placeholder texts as their styling often results in text that is harder to read. But that is up to you to decide. 

Please check our coding examples about how to write accessible forms. [Maybe this guide is also helpful for you.](https://developer.mozilla.org/en-US/docs/Learn/Forms/How_to_structure_a_web_form)
You should also wrap your input element in a form element as every search is also a form.

Reply:
This is the next step for my coming userstory. It was only to test the link for me....

Reply:
Ok! 👍